src/ucp/api/ucp.h

```diff
@@ -2569,12 +2569,15 @@ ucs_status_ptr_t ucp_tag_send_sync_nb(ucp_ep_h ep, const void *buffer, size_t co
  * The routine is non-blocking and therefore returns immediately, however the
  * actual send operation may be delayed. The send operation is considered
  * completed when it is safe to reuse the source @e buffer. If the send
- * operation is completed immediately the routine return UCS_OK and the
+ * operation is completed immediately the routine returns UCS_OK and the
  * call-back function is @b not invoked. If the operation is @b not completed
  * immediately and no error reported then the UCP library will schedule to
- * invoke the call-back whenever the send operation will be completed. In other
+ * invoke the call-back whenever the send operation is completed. In other
  * words, the completion of a message can be signaled by the return code or
  * the call-back.
+ * Immediate completion signals can be fine-tuned via the op_attr_mask
```

Comment:
I don't think you added this to the right place. This new clarification is for new functions that accept the request parameter

Reply:
didn't get... what function do you mean?

Reply:
@shamisp  per https://github.com/openucx/ucx/pull/4784#discussion_r396627541 ,we are documenting ucp_request_param_t

Reply:
@shamisp the comment is for ucp_tag_send_nbx, which is this new function

Reply:
@yosefe it shows differently in my diff

Reply:
@hoopoepg I see it correctly now. `@ref` to structure, structure member, and  enumerate are missing.